packages/json-extension/src/component.tsx

```diff
@@ -21,6 +21,11 @@ export interface IProps {
    * The application language translator.
    */
   translator?: ITranslator;
+
+  /**
+   * Function called after the component is rendered
+   */
+  callback?: () => void;
```

Comment:
Could you use `React.forwardRef` to avoid using something specific that may be cryptic of common React developer: https://reactjs.org/docs/forwarding-refs.html#forwarding-refs-to-dom-components